libnetwork/internal/rootlessnetns/netns_linux.go

```diff
@@ -113,7 +114,14 @@ func (n *Netns) getOrCreateNetns() (ns.NetNS, bool, error) {
 	if err != nil {
 		return nil, false, wrapError("create netns", err)
 	}
-	err = n.setupSlirp4netns(nsPath)
+	switch strings.ToLower(n.config.Network.DefaultRootlessNetworkCmd) {
+	case "", slirp4netns.BinaryName:
```

Comment:
Should this be switched to default to pasta?

Reply:
I don't think it matters, it should always be explicitly set in containers.conf

Reply:
Some distributions do not ship a containers.conf, or was the default modified in common/pkg/config/default.go?

Reply:
Looks like not yet.

			DefaultRootlessNetworkCmd: "slirp4netns",